npm/ng-packs/packages/components/extensible/src/lib/components/extensible-table/extensible-table.component.ts

```diff
@@ -207,7 +207,7 @@ export class ExtensibleTableComponent<R = any> implements OnChanges {
   }
 
   hasAvailableActions(index, data): boolean {
-    const { permission, visible } = this.actionList.get(index).value;
+    const { permission, visible } = (this.actionList.get(index) || {}).value || {};
```

Comment:
```suggestion
    const { permission, visible } = this.actionList.get(index)?.value || {};
```

Reply:
@sumeyyeKurtulus can you please apply like this